docs/getting-started-guides/openstack-heat.md

```diff
@@ -0,0 +1,239 @@
+---
+---
+
+* TOC
+{:toc}
```

Comment:
This is some github.io rendering magic: http://elsonrodriguez.github.io/docs/getting-started-guides/openstack-heat/

Might need some guidance on how to get the navigation on the left hand side appear, along with the Header: http://elsonrodriguez.github.io/docs/getting-started-guides/aws/